x/params/store/store.go

```diff
@@ -0,0 +1,171 @@
+package store
+
```

Comment:
Are there any unit tests for these?

Reply:
The tests for store.go are contained in `../keeper_test.go` - it verifies the methods of `Store`